WISE-Community/WISE-Client
# feat(QuestionBank): Add authoring support

## Changes
- Allow author to add/delete/order/edit QuestionBank items, similar to FeedbackRule and DynamicPrompt. QuestionBank allows author to specify one or more questions per rule, similar to v2 of FeedbackRule
- Hide QuestionBank from student/grading view if it is not disabled

## Test Prep
- If you have any existing PeerChat component, be sure to remove the ```questionsBank``` field from the component content's JSON if it's an empty array. The new format is a JSON object

## Test
- Add/delete/order/edit QuestionBank
- Add/delete QuestionBank question
- Enable/disable QuestionBank should show/hide QuestionBank from student/grading views

Closes #911
